File: client/src/models/Account.js
export class Account {
  constructor(data) {
    this.id = data.id
    this.email = data.email
    this.name = data.name
    this.picture = data.picture
    this.wantsCelsius = data.wantsCelsius
    this.wantsTwentyFourClock = data.wantsTwentyFourClock
    this.preferredImageTypes = data.preferredImageTypes
    this.preferredLocation = data.preferredLocation
    // TODO add additional properties if needed
  }
}

File: server/Models/Account.cs
namespace inspireVue.Models;

public class Account
{
  public string Id { get; set; }
  public string Name { get; set; }
  public string Email { get; set; }
  public string Picture { get; set; }

  public bool? WantsCelsius { get; set; }

  public bool? WantsTwentyFourClock { get; set; }

  public string PreferredImageTypes { get; set; }
}


// testing testing again